pkg/tagger/tagger.go

```diff
@@ -213,6 +214,34 @@ func (t *Tagger) Stop() error {
 	return nil
 }
 
+// OutdatedTags returns true if an ADIdentifiers contains is not up to date
+func (t *Tagger) OutdatedTags(ADIdentifiers []string) bool {
```

Comment:
Is there a reason to not leave that logic on the `TagStore` itself? This allows for easier testing, by manipulating the simpler `TagStore` item instead of the whole `Tagger`